examples/GreeterProtobuf/Server/Program.cs

```diff
@@ -3,9 +3,8 @@
 using GreeterProtobufServer;
 using IceRpc;
 
-// Create a server that dispatches all requests to the same service, an instance of Chatbot.
+// Create a server that will dispatch all requests to the same service, an instance of Chatbot.
```

Comment:
Please keep present tense.

Reply:
That comes from our Greeter example https://github.com/icerpc/icerpc-csharp/blob/06ceee30fbeced504a32d2c0a068d326daf063d8/examples/Greeter/Server/Program.cs#L6

Reply:
I would fix the other one.